dev-manuals/architecture/chickenshock-backend/README.md

```diff
@@ -0,0 +1,45 @@
+# Architecture of Chickenshock-backend
+
+Project: <https://github.com/Gamify-IT/chickenshock-backend> \
+Scripts: <https://github.com/Gamify-IT/chickenshock-backend/tree/main/src>
+
+## Purpose
+
+This backend saves relevant Chickenshock game data.
+
+## General code structure
+
+# Architecture of Chickenshock-Backend,
+
+This backend has the following package structure
+- `data`: Contains the data objects and DTOs
+- `controllers`: Containers the server routes
+- `services`: Contains the business logic
+- `clients`: Contains the b2b (backend-2-backend) communication methods
+- `repositories`: Contains the database repositories
+
+
+## Starting the service
+
+See the README or the user manual.
+
+## What to look at
+
+- every data-class has an additional DTO class for savety purposes
```

Comment:
```suggestion
- every entity class has an additional DTO class for safety purposes
```

Reply:
Why explicitly look at the DTOs?

Reply:
See also https://github.com/Gamify-IT/docs/pull/128#discussion_r955134299

Reply:
idk i thought it was atleast a little bit special... we can remove it if you want

Reply:
oh ok it already is removed